collectors/graph_collector.go

```diff
@@ -366,6 +402,27 @@ func (g *GraphCollector) collectRoutingPolicyMetrics(
 
 		feeBaseStats = newStatsCompiler(numEdges)
 		feeRateStats = newStatsCompiler(numEdges)
+
+		inboundFeeBaseStats = prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name: inboundFeeBaseName,
+				Help: inboundFeeBaseHelp,
```

Comment:
Ah as attributes? I think the constants fit better since they bind the name, label, etc.

Reply:
I was more looking at the pattern that we use for other values
```go
	ch <- prometheus.MustNewConstMetric(
		g.minTimelockDeltaDesc, prometheus.GaugeValue,
		timelockReport.min,
	)
```
where we take the description which is defined in a single place in `NewGraphCollector`, but in this PR the constants are referenced in two places. We could use `prometheus.MustNewConstHistogram` to follow that pattern, but that seems to be harder to use since we would need to keep track of buckets ourselves.